exercises/grade-school/grade_school_test.rb

```diff
@@ -4,85 +4,104 @@
 require_relative 'grade_school'
 
 class SchoolTest < Minitest::Test
-  attr_reader :school
-
-  def setup
-    @school = School.new
-  end
-
-  def test_an_empty_school
-    assert_equal({}, school.to_h)
+  def test_empty_grade
+    subject = School.new
+    expected = []
+    assert_equal expected, subject.grade(1)
   end
 
   def test_add_student
-    skip
-    school.add('Aimee', 2)
-    assert_equal({ 2 => ['Aimee'] }, school.to_h)
+    subject = School.new
+    assert subject.add('Aimee', 2)
+    expected = ['Aimee']
+    assert_equal expected, subject.grade(2)
   end
 
-  def test_add_more_students_in_same_class
+  def test_add_students_to_different_grades
     skip
-    school.add('Blair', 2)
-    school.add('James', 2)
-    school.add('Paul', 2)
-    assert_equal({ 2 => %w(Blair James Paul) }, school.to_h)
+    subject = School.new
+    subject.add('Aimee', 3)
+    subject.add('Beemee', 7)
+    assert_equal ['Aimee'], subject.grade(3)
+    assert_equal ['Beemee'], subject.grade(7)
   end
 
-  def test_add_students_to_different_grades
+  def test_grade_with_multiple_students
     skip
-    school.add('Chelsea', 3)
-    school.add('Logan', 7)
-    assert_equal({ 3 => ['Chelsea'], 7 => ['Logan'] }, school.to_h)
+    subject  = School.new
+    grade    = rand(6)
```

Comment:
This will allow `0`, which is a bit odd. Any particular reason to have random grades?